simulation_parameters/SimulationParameters.cs

```diff
@@ -177,17 +178,16 @@ public bool DoesCompoundExist(string name)
     /// </summary>
     public List<Compound> GetCloudCompounds()
     {
-        var result = new List<Compound>();
-
-        foreach (var entry in compounds)
-        {
-            if (entry.Value.IsCloud)
-            {
-                result.Add(entry.Value);
-            }
-        }
+        return compounds.Values.Where(c => c.IsCloud).ToList();
+    }
 
-        return result;
+    /// <summary>
+    ///   Returns all environmental *molecular* compounds that are dissolved in the environment, i.e. gas.
+    /// </summary>
+    /// <remarks>This excludes sunlight, and includes O2, CO2, N...</remarks>
```

Comment:
remarks needs to have a para tag in it and it needs to be formatted like shown in the styleguide.